data_steward/tools/run_deid.py

```diff
@@ -291,6 +291,34 @@ def load_deid_map_table(client, deid_map_dataset_name, age_limit):
         )
 
 
+def copy_ext_tables(bq_client, input_dataset: str, output_dataset: str):
+    """
+    Copy extension tables to the deid dataset
+
+    :param bq_client: a BigQueryClient
+    :param input_dataset: Name of the input dataset
+    :param output_dataset: Name of the output dataset
+    :return: job_list: list of job_ids
+    """
+    source_tables = bq_client.list_tables(
+        f'{bq_client.project}.{input_dataset}')
+    job_config = CopyJobConfig(write_disposition=WriteDisposition.WRITE_EMPTY)
+    job_list = []
+    for table in source_tables:
+        if table.table_id.endswith('_ext'):
+            destination_table = f'{output_dataset}.{table.table_id}'
+            job_config.labels.update({
+                'table_name': table.table_id.lower(),
+                'copy_from': input_dataset.lower(),
+                'copy_to': output_dataset.lower()
+            })
```

Comment:
This is the second or third time I've seen these copy job labels.  Does it make sense to move this to a utility somewhere?  What are your recommendations?

Reply:
There is a ticket thay i am working on which fixes the illegal characters in the lables which touches both the places you mentioned, do you think, it will be good ideat to refactor there instead of here?

Reply:
That does sound like a better place to make this change.